.eslintrc

```diff
@@ -118,6 +135,50 @@
 
     "flowtype/define-flow-type": 1,
     "flowtype/use-flow-type": 1,
-    "flowtype/space-after-type-colon": [1, "always"]
+    "flowtype/space-after-type-colon": [1, "always"],
+
+    // This makes sure imported modules exist.
+    "import/no-unresolved": ["error"],
```

Comment:
@shubheksha in the other rules configured in this .eslintrc file, we currently use `2` in place of "error", we should probably keep it consistent with this new group of rules as well.

(Also, `"rule": ["error"]` can be probably be rewritten to `"rule": 2` for the same reason)